android-client-sdk/src/main/java/com/devcycle/sdk/android/util/DVCSharedPrefs.kt

```diff
@@ -10,9 +10,10 @@ import com.fasterxml.jackson.core.JsonProcessingException
 import com.fasterxml.jackson.core.type.TypeReference
 import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
 import timber.log.Timber
+import java.util.Calendar
 
 // TODO: access disk on background thread
-class DVCSharedPrefs(context: Context) {
+internal class DVCSharedPrefs(context: Context) {
```

Comment:
why do we need to make this class internal?

Reply:
the PopulatedUser class is an internal class, and a public class can't accept an internal class as a parameter — but also, the DVCSharedPrefs shouldn't be publicly accessible